src/main/kotlin/no/nav/helse/spion/vedtaksmelding/VedtaksmeldingService.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.helse.spion.vedtaksmelding
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import no.nav.helse.spion.domene.Arbeidsgiver
+import no.nav.helse.spion.domene.Periode
+import no.nav.helse.spion.domene.Person
+import no.nav.helse.spion.domene.ytelsesperiode.Arbeidsforhold
+import no.nav.helse.spion.domene.ytelsesperiode.Ytelsesperiode
+import no.nav.helse.spion.domene.ytelsesperiode.repository.YtelsesperiodeRepository
+import java.time.LocalDate
+
+class VedtaksmeldingService(
+        private val ypRepo: YtelsesperiodeRepository,
+        private val om: ObjectMapper
+) {
+
+    fun processAndSaveMessage(melding: String) {
+        val deserializedKafkaMessage = om.readValue(melding, Vedtaksmelding::class.java)
+        val mapped = mapVedtaksMeldingTilYtelsesPeriode(deserializedKafkaMessage)
+        ypRepo.save(mapped)
+    }
+
+    companion object Mapper {
+        fun mapVedtaksMeldingTilYtelsesPeriode(vm: Vedtaksmelding): Ytelsesperiode {
```

Comment:
Er jo noen hardkodede verdier her, men de er kanskje utilgjengelige akkurat nå?

Reply:
Jepp, de er placeholdere for ting vi ikke vet om vi får eller om vi skal ha